lib/ZumoOta/src/App.cpp

```diff
@@ -74,45 +83,66 @@ App::~App()
 {
 }
 
-void App::start() 
+bool App::loginit()
+{
+    if (true == Logging::getInstance().registerSink(&gLogSinkSerial))
+    {
+        (void)Logging::getInstance().selectSink("Serial");
+
+        /* Set severity of logging system. */
+        Logging::getInstance().setLogLevel(CONFIG_LOG_SEVERITY);
+
+        LOG_DEBUG("LOGGER READY");
+        return true;
+    }
+    else 
+    {
+        LOG_ERROR("Fail to init Logging!");

```

Comment:
Does this have any value if the sink register failed in line 88 ? Guess it doesn't hurt either, but expect logging to not work. I would delete the else.

Reply:
It will be taken care of by the next commit.